types/aws-lambda/trigger/appsync-resolver.d.ts

```diff
@@ -1,20 +1,44 @@
 import { Handler } from '../handler';
 
-export type AppSyncResolverHandler<TArguments, TResults, TSource = Record<string, any> | null> = Handler<AppSyncResolverEvent<TArguments, TSource>, TResults | TResults[]>;
+export type AppSyncResolverHandler<TArguments, TResults, TSource = Record<string, any> | null> = Handler<
+    AppSyncResolverEvent<TArguments, TSource>,
+    TResults
```

Comment:
@bboure Should `TResults` be `TResult` and below `TResults[]` be `TResult`?

Reply:
Thanks @jakejscott

I thought the same but I actually hesitated to change it. I finally kept it as it was.
This is not a big deal though and will not break anything. I'll change it